dts/bindings/spi/litex,spi0.yaml

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2019 Antmicro <www.antmicro.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+title: LiteX SPI
+
+description: >
+    This binding gives a base representation of the LiteX SPI
+
+compatible: "litex,spi0"
```

Comment:
Why is this `spi0` instead of `spi`? This looks like an instance number, which shouldn't be in the compatible

Reply:
I changed compatible to "litex,spi" and renamed the whole file to "litex,spi.yaml"